mon-pix/app/components/reached-stage.js

```diff
@@ -0,0 +1,16 @@
+import Component from '@glimmer/component';
+
+export default class ReachedStage extends Component {
+
+  get plainStars() {
+    // returns an array of [1...N] where N is the number of plain stars
+    return Array.from(Array(this.args.starCount), (element ,index) => index + 1);
```

Comment:
Peut-être qu'on gagnerait à encapsuler ça dans une variable bien nommée ?
...
En cherchant à comprendre ce que faisait Array.from, j'ai vu que IE y était intolérant 
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/from

Reply:
Proposition : _.range(1, this.args.starCount +1)